docs/finalproject/external-projects.md

```diff
@@ -0,0 +1,127 @@
+---
+id: external-projects
+title: External Projects
+sidebar_label: External Projects
+---
+
+At CodeYourFuture we try to find interesting, tech-for-good projects for that stretch the abilities of our students whilst helping an NGO or Charity solve a business problem they are facing.
+
+## Past Projects
+
+Here are some of the projects that we've worked on in the past
+
+- [Over The Line with The Compass Collective](https://over-the-line.uk/)
+
+Additionally, here are some examples of internal projects we've completed
+
+- [Class Planner](https://cyf-class-planner.herokuapp.com/)
+
+You can also find more example project briefs on the Project Brief
+
+## Goals
+
+With the final projects, we’re looking to give our students:
+
+- Experience of collaborating in a team to deliver software
+- An opportunity to solve real world business problems
+- The chance to bring together the different things they’ve learned
+- A real example to talk about in job interviews and guide their interests
+
+and to give our partners:
+
+- A useful piece of software that they can use as-is or build on after the project
+
+More details can be found on the [Intro](./intro) and [Preparation](./prep) pages.
+
+## What does a Product Owner have to do?
+
+To be a good Stakeholder/Product Owner you should
+
+- Be able to attend a weekly/bi-weekly meeting with our teams
+- Be in a position to advise the development of the project
+  - e.g. Should we prioritise Feature A or Feature B?
```

Comment:
Perhaps this covers it already, but maybe something like "Be willing to cut scope" (or an equivalent non-jargon phrase)?